### PR TITLE
DT-1490: ⚡️ Add in search filter for active / inactivate users

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: sass/_horizontal-content.scss [hunk 1]
```diff
@@ -0,0 +1,88 @@
+.horizontal-information {
```
Comment (petergphillips) [inline sass/_horizontal-content.scss:1]: grabbed from prison staff hub

File: sass/application.scss [hunk 2]
```diff
@@ -71,25 +71,6 @@ $govuk-page-width: 1170px;
   }
 }
 
-.horizontal-links {
```
Comment (petergphillips) [inline sass/application.scss:74]: moved into horizontal sass file

File: views/addGroup.njk [hunk 3]
```diff
@@ -42,6 +42,7 @@
           {{ govukButton({
             text: "Confirm",
             type: "submit",
+            classes: "govuk-!-margin-right-1",
```
Comment (petergphillips) [inline views/addGroup.njk:45]: found in govuk design system that their buttons were better spaced because they had added this to all of them

**Review by Mjwillis — APPROVED** [review]
LGTM

**Pull request merged**